src/Toolkit/Toolkit.Maui/BasemapGallery/BasemapGallery.cs

```diff
@@ -44,16 +44,23 @@ public BasemapGallery()
         Loaded += BasemapGallery_Loaded;
     }
 
-    private async void BasemapGallery_Loaded(object? sender, EventArgs e)
-    {
-        // Unsubscribe from the Loaded event to ensure this only runs once.
-        Loaded -= BasemapGallery_Loaded;
-
-        if (AvailableBasemaps is null)
-        {
-            await _controller.LoadFromDefaultPortal();
-        }
-    }
+    private async void BasemapGallery_Loaded(object? sender, EventArgs e)
+    {
+        try
+        {
+            // Unsubscribe from the Loaded event to ensure this only runs once.
+            Loaded -= BasemapGallery_Loaded;
+
+            if (AvailableBasemaps is null)
+            {
+                await _controller.LoadFromDefaultPortal();
+            }
+        }
+        catch (Exception ex)
+        {
+            System.Diagnostics.Debug.WriteLine($"Error loading basemaps: {ex.Message}");
```

Comment:
Debug.WriteLine doesn't work in release build of the API which is all we ship. You will need Trace.WriteLine for that.
However we should already be writing out load errors to the debug console.